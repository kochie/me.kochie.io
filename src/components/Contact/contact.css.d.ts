export const contactInfo: string;
export const name: string;
export const email: string;
export const button: string;
export const icon: string;
export const fit: string;
export const small: string;
export const big: string;
export const special: string;
export const disabled: string;
export const form: string;
export const selectWrapper: string;
export const formerizePlaceholder: string;
