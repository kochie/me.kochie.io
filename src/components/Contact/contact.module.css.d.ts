export const contactInfo: string;
export const name: string;
export const email: string;
