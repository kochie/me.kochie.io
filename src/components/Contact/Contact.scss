@import '../../styles/constants';

.contact-info {
  display: flex;
	flex-direction: row;
	justify-content: space-between;
	@media only screen and (max-width: $mobile-width) and (max-height: $mobile-height) and (min-resolution: $mobile-resolution),
	only screen and (max-width: $mobile-height) and (max-height: $mobile-width) and (min-resolution: $mobile-resolution) {
		flex-direction: column;
		align-items: center;
	}
}

.name {
	margin-bottom: 2%;
	margin-right: 2%;
	@media only screen and (max-width: $mobile-width) and (max-height: $mobile-height) and (min-resolution: $mobile-resolution),
	only screen and (max-width: $mobile-height) and (max-height: $mobile-width) and (min-resolution: $mobile-resolution) {
		margin: 2%;
		box-sizing: border-box;
	}
}

.email {
	margin-bottom: 2%;
	@media only screen and (max-width: $mobile-width) and (max-height: $mobile-height) and (min-resolution: $mobile-resolution),
	only screen and (max-width: $mobile-height) and (max-height: $mobile-width) and (min-resolution: $mobile-resolution) {
		margin: 0 2% 2% 2%;
		box-sizing: border-box;
	}
}

$button-colour: #7998a5;

input[type="submit"],
input[type="reset"],
input[type="button"],
.button {
-moz-appearance: none;
-webkit-appearance: none;
-ms-appearance: none;
appearance: none;
-moz-transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
-webkit-transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
-ms-transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
background-color: transparent;
border-radius: 0.35em;
border: solid 3px #efefef;
color: #787878;
cursor: pointer;
display: inline-block;
font-weight: 400;
height: 3.15em;
height: calc(2.75em + 6px);
line-height: 2.75em;
min-width: 10em;
padding: 0 1.5em;
text-align: center;
text-decoration: none;
white-space: nowrap;
}

input[type="submit"]:hover,
input[type="reset"]:hover,
input[type="button"]:hover,
.button:hover {
  border-color: $button-colour;
  color: $button-colour;
}

input[type="submit"]:active,
input[type="reset"]:active,
input[type="button"]:active,
.button:active {
  background-color: rgba(73, 191, 157, 0.1);
  border-color: $button-colour;
  color: $button-colour;
}

input[type="submit"].icon,
input[type="reset"].icon,
input[type="button"].icon,
.button.icon {
  padding-left: 1.35em;
}

  input[type="submit"].icon:before,
  input[type="reset"].icon:before,
  input[type="button"].icon:before,
  .button.icon:before {
    margin-right: 0.5em;
  }

input[type="submit"].fit,
input[type="reset"].fit,
input[type="button"].fit,
.button.fit {
  display: block;
  margin: 0 0 1em 0;
  min-width: 0;
  width: 100%;
}

input[type="submit"].small,
input[type="reset"].small,
input[type="button"].small,
.button.small {
  font-size: 0.8em;
}

input[type="submit"].big,
input[type="reset"].big,
input[type="button"].big,
.button.big {
  font-size: 1.35em;
}

input[type="submit"].special,
input[type="reset"].special,
input[type="button"].special,
.button.special {
  background-color: $button-colour;
  border-color: $button-colour;
  color: #ffffff;
}

  input[type="submit"].special:hover,
  input[type="reset"].special:hover,
  input[type="button"].special:hover,
  .button.special:hover {
    background-color: $button-colour;
    border-color: $button-colour;
  }

  input[type="submit"].special:active,
  input[type="reset"].special:active,
  input[type="button"].special:active,
  .button.special:active {
    background-color: $button-colour;
    border-color: $button-colour;
  }

input[type="submit"].disabled, input[type="submit"]:disabled,
input[type="reset"].disabled,
input[type="reset"]:disabled,
input[type="button"].disabled,
input[type="button"]:disabled,
.button.disabled,
.button:disabled {
  background-color: #e7e7e7 !important;
  border-color: #e7e7e7 !important;
  color: #b2b2b2 !important;
  cursor: default;
}

.form {
  form {
		margin: 0 0 2em 0;
	}

	label {
		color: #787878;
		display: block;
		font-size: 0.9em;
		font-weight: 400;
		margin: 0 0 1em 0;
	}

	input[type="text"],
	input[type="password"],
	input[type="email"],
	select,
	textarea {
		-moz-appearance: none;
		-webkit-appearance: none;
		-ms-appearance: none;
		appearance: none;
		background: #ededed;
		border-radius: 0.35em;
		border: solid 2px transparent;
		color: inherit;
		display: block;
		outline: 0;
		padding: 0 0.75em;
		text-decoration: none;
		width: 100%;
	}

		input[type="text"]:invalid,
		input[type="password"]:invalid,
		input[type="email"]:invalid,
		select:invalid,
		textarea:invalid {
			box-shadow: none;
		}

		input[type="text"]:focus,
		input[type="password"]:focus,
		input[type="email"]:focus,
		select:focus,
		textarea:focus {
			border-color: $button-colour;
		}

	.select-wrapper {
		text-decoration: none;
		display: block;
		position: relative;
	}

		.select-wrapper:before {
			-moz-osx-font-smoothing: grayscale;
			-webkit-font-smoothing: antialiased;
			font-family: FontAwesome;
			font-style: normal;
			font-weight: normal;
			text-transform: none !important;
		}

		.select-wrapper:before {
			color: #dfdfdf;
			content: '\f078';
			display: block;
			height: 2.75em;
			line-height: 2.75em;
			pointer-events: none;
			position: absolute;
			right: 0;
			text-align: center;
			top: 0;
			width: 2.75em;
		}

		.select-wrapper select::-ms-expand {
			display: none;
		}

	input[type="text"],
	input[type="password"],
	input[type="email"],
	select {
		height: 2.75em;
	}

	textarea {
		padding: 0.75em;
		font-family: 'Source Sans Pro';
		box-sizing: border-box;
	}

	input[type="checkbox"],
	input[type="radio"] {
		-moz-appearance: none;
		-webkit-appearance: none;
		-ms-appearance: none;
		appearance: none;
		display: block;
		float: left;
		margin-right: -2em;
		opacity: 0;
		width: 1em;
		z-index: -1;
	}

		input[type="checkbox"] + label,
		input[type="radio"] + label {
			text-decoration: none;
			color: #a2a2a2;
			cursor: pointer;
			display: inline-block;
			font-size: 1em;
			font-weight: 400;
			padding-left: 2.4em;
			padding-right: 0.75em;
			position: relative;
		}

			input[type="checkbox"] + label:before,
			input[type="radio"] + label:before {
				-moz-osx-font-smoothing: grayscale;
				-webkit-font-smoothing: antialiased;
				font-family: FontAwesome;
				font-style: normal;
				font-weight: normal;
				text-transform: none !important;
			}

			input[type="checkbox"] + label:before,
			input[type="radio"] + label:before {
				background: #f7f7f7;
				border-radius: 0.35em;
				border: solid 2px transparent;
				content: '';
				display: inline-block;
				height: 1.65em;
				left: 0;
				line-height: 1.58125em;
				position: absolute;
				text-align: center;
				top: 0;
				width: 1.65em;
			}

		input[type="checkbox"]:checked + label:before,
		input[type="radio"]:checked + label:before {
			background: #787878;
			border-color: #787878;
			color: #fff;
			content: '\f00c';
		}

		input[type="checkbox"]:focus + label:before,
		input[type="radio"]:focus + label:before {
			border-color: $button-colour;
		}

	input[type="checkbox"] + label:before {
		border-radius: 0.35em;
	}

	input[type="radio"] + label:before {
		border-radius: 100%;
	}

	::-webkit-input-placeholder {
		color: #b2b2b2 !important;
		opacity: 1.0;
	}

	:-moz-placeholder {
		color: #b2b2b2 !important;
		opacity: 1.0;
	}

	::-moz-placeholder {
		color: #b2b2b2 !important;
		opacity: 1.0;
	}

	:-ms-input-placeholder {
		color: #b2b2b2 !important;
		opacity: 1.0;
	}

	.formerize-placeholder {
		color: #b2b2b2 !important;
		opacity: 1.0;
	}
}