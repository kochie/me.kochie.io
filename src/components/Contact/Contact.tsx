import React, { useState } from "react";

import { ContactButton } from "..";

import ContactInfo from "./contact.mdx";
import {
  contactInfo,
  name as nameStyle,
  email as emailStyle
} from "./contact.module.css";

export default () => {
  const [name, setName] = useState("");
  const [email, setEmail] = useState("");
  const [message, setMessage] = useState("");

  return (
    <div>
      <h2>Contact</h2>
      <ContactInfo />
      <div id="map" />
      <form action="https://formspree.io/robert@kochie.io" method="POST">
        <div>
          <div className={contactInfo}>
            <input
              className={nameStyle}
              type="text"
              placeholder="Name"
              name="name"
              value={name}
              onChange={e => setName(e.target.value)}
            />
            <input
              className={emailStyle}
              type="text"
              placeholder="Email"
              name="email"
              value={email}
              onChange={e => setEmail(e.target.value)}
            />
          </div>
          <div className="message">
            <textarea
              placeholder="Message"
              name="message"
              rows={4}
              value={message}
              onChange={e => setMessage(e.target.value)}
            />
          </div>
        </div>
        <ContactButton message={message} name={name} />
      </form>
    </div>
  );
};
