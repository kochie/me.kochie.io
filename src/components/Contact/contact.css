@media (--mobile-size) {
  .contact-info {
    flex-direction: column;
    align-items: center;
  }
  .name {
    margin: 2%;
    box-sizing: border-box;
  }
  .email {
    margin: 0 2% 2% 2%;
    box-sizing: border-box;
  }
}

.contact-info {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
}

.name {
  margin-bottom: 2%;
  margin-right: 2%;
}

.email {
  margin-bottom: 2%;
}

:root {
  --button-colour: #7998a5;
  --inactive-colour: #b2b2b2;
}

input[type="submit"],
input[type="reset"],
input[type="button"],
.button {
  appearance: none;
  transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out,
    border-color 0.2s ease-in-out;
  background-color: transparent;
  border-radius: 0.35em;
  border: solid 3px --inactive-colour;
  color: --inactive-colour;
  cursor: pointer;
  display: inline-block;
  font-weight: 400;
  height: 3.15em;
  height: calc(2.75em + 6px);
  line-height: 2.75em;
  min-width: 10em;
  padding: 0 1.5em;
  text-align: center;
  text-decoration: none;
  white-space: nowrap;
}

input[type="submit"]:hover,
input[type="reset"]:hover,
input[type="button"]:hover,
.button:hover {
  border-color: --button-colour;
  color: --button-colour;
}

input[type="submit"]:active,
input[type="reset"]:active,
input[type="button"]:active,
.button:active {
  background-color: rgba(73, 191, 157, 0.1);
  border-color: --button-colour;
  color: --button-colour;
}

input[type="submit"].icon,
input[type="reset"].icon,
input[type="button"].icon,
.button.icon {
  padding-left: 1.35em;
}

input[type="submit"].icon:before,
input[type="reset"].icon:before,
input[type="button"].icon:before,
.button.icon:before {
  margin-right: 0.5em;
}

input[type="submit"].fit,
input[type="reset"].fit,
input[type="button"].fit,
.button.fit {
  display: block;
  margin: 0 0 1em 0;
  min-width: 0;
  width: 100%;
}

input[type="submit"].small,
input[type="reset"].small,
input[type="button"].small,
.button.small {
  font-size: 0.8em;
}

input[type="submit"].big,
input[type="reset"].big,
input[type="button"].big,
.button.big {
  font-size: 1.35em;
}

input[type="submit"].special,
input[type="reset"].special,
input[type="button"].special,
.button.special {
  background-color: --button-colour;
  border-color: --button-colour;
  color: #ffffff;
}

input[type="submit"].special:hover,
input[type="reset"].special:hover,
input[type="button"].special:hover,
.button.special:hover {
  background-color: --button-colour;
  border-color: --button-colour;
}

input[type="submit"].special:active,
input[type="reset"].special:active,
input[type="button"].special:active,
.button.special:active {
  background-color: --button-colour;
  border-color: --button-colour;
}

input[type="submit"].disabled,
input[type="submit"]:disabled,
input[type="reset"].disabled,
input[type="reset"]:disabled,
input[type="button"].disabled,
input[type="button"]:disabled,
.button.disabled,
.button:disabled {
  background-color: #e7e7e7 !important;
  border-color: #e7e7e7 !important;
  color: #b2b2b2 !important;
  cursor: default;
}

.form form {
  margin: 0 0 2em 0;
}

.form label {
  color: #787878;
  display: block;
  font-size: 0.9em;
  font-weight: 400;
  margin: 0 0 1em 0;
}

.form input[type="text"],
.form input[type="password"],
.form input[type="email"],
.form select,
.form textarea {
  -moz-appearance: none;
  -webkit-appearance: none;
  -ms-appearance: none;
  appearance: none;
  background: #ededed;
  border-radius: 0.35em;
  border: solid 2px transparent;
  color: inherit;
  display: block;
  outline: 0;
  padding: 0 0.75em;
  text-decoration: none;
  width: 100%;
}

.form input[type="text"]:invalid,
.form input[type="password"]:invalid,
.form input[type="email"]:invalid,
.form select:invalid,
.form textarea:invalid {
  box-shadow: none;
}

.form input[type="text"]:focus,
.form input[type="password"]:focus,
.form input[type="email"]:focus,
.form select:focus,
.form textarea:focus {
  border-color: --button-colour;
}

.form .select-wrapper {
  text-decoration: none;
  display: block;
  position: relative;
}

.form .select-wrapper:before {
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  font-family: FontAwesome;
  font-style: normal;
  font-weight: normal;
  text-transform: none !important;
}

.form .select-wrapper:before {
  color: #dfdfdf;
  content: "\f078";
  display: block;
  height: 2.75em;
  line-height: 2.75em;
  pointer-events: none;
  position: absolute;
  right: 0;
  text-align: center;
  top: 0;
  width: 2.75em;
}

.form .select-wrapper select::-ms-expand {
  display: none;
}

.form input[type="text"],
.form input[type="password"],
.form input[type="email"],
.form select {
  height: 2.75em;
}

.form textarea {
  padding: 0.75em;
  font-family: "Source Sans Pro";
  box-sizing: border-box;
}

.form input[type="checkbox"],
.form input[type="radio"] {
  -moz-appearance: none;
  -webkit-appearance: none;
  -ms-appearance: none;
  appearance: none;
  display: block;
  float: left;
  margin-right: -2em;
  opacity: 0;
  width: 1em;
  z-index: -1;
}

.form input[type="checkbox"] + label,
.form input[type="radio"] + label {
  text-decoration: none;
  color: #a2a2a2;
  cursor: pointer;
  display: inline-block;
  font-size: 1em;
  font-weight: 400;
  padding-left: 2.4em;
  padding-right: 0.75em;
  position: relative;
}

.form input[type="checkbox"] + label:before,
.form input[type="radio"] + label:before {
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  font-family: FontAwesome;
  font-style: normal;
  font-weight: normal;
  text-transform: none !important;
}

.form input[type="checkbox"] + label:before,
.form input[type="radio"] + label:before {
  background: #f7f7f7;
  border-radius: 0.35em;
  border: solid 2px transparent;
  content: "";
  display: inline-block;
  height: 1.65em;
  left: 0;
  line-height: 1.58125em;
  position: absolute;
  text-align: center;
  top: 0;
  width: 1.65em;
}

.form input[type="checkbox"]:checked + label:before,
.form input[type="radio"]:checked + label:before {
  background: #787878;
  border-color: #787878;
  color: #fff;
  content: "\f00c";
}

.form input[type="checkbox"]:focus + label:before,
.form input[type="radio"]:focus + label:before {
  border-color: --button-colour;
}

.form input[type="checkbox"] + label:before {
  border-radius: 0.35em;
}

.form input[type="radio"] + label:before {
  border-radius: 100%;
}

.form ::-webkit-input-placeholder {
  color: #b2b2b2 !important;
  opacity: 1;
}

.form :-moz-placeholder {
  color: #b2b2b2 !important;
  opacity: 1;
}

.form ::-moz-placeholder {
  color: #b2b2b2 !important;
  opacity: 1;
}

.form :-ms-input-placeholder {
  color: #b2b2b2 !important;
  opacity: 1;
}

.form .formerize-placeholder {
  color: #b2b2b2 !important;
  opacity: 1;
}
