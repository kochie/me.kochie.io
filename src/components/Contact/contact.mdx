If you'd like to get in contact, feel free to write a message or send an email. You can also grab a copy of my resume below.