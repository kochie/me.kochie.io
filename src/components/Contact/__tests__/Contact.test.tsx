import React from "react";
import renderer from "react-test-renderer";

import Contact from "../Contact";

import { fab } from "@fortawesome/free-brands-svg-icons";
import { library } from "@fortawesome/fontawesome-svg-core";
library.add(fab);

it("contact renders correctly", () => {
  const tree = renderer.create(<Contact />).toJSON();
  expect(tree).toMatchSnapshot();
});
