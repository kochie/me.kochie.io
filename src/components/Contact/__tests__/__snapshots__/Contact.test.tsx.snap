// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact renders correctly 1`] = `
<div>
  <h2>
    Contact
  </h2>
  <p>
    If you'd like to get in contact, feel free to write a message or send an email. You can also grab a copy of my resume below.
  </p>
  <div
    id="map"
  />
  <form
    action="https://formspree.io/robert@kochie.io"
    method="POST"
  >
    <div>
      <div
        className="contactInfo"
      >
        <input
          className="name"
          name="name"
          onChange={[Function]}
          placeholder="Name"
          type="text"
          value=""
        />
        <input
          className="email"
          name="email"
          onChange={[Function]}
          placeholder="Email"
          type="text"
          value=""
        />
      </div>
      <div
        className="message"
      >
        <textarea
          name="message"
          onChange={[Function]}
          placeholder="Message"
          rows={4}
          value=""
        />
      </div>
    </div>
    <div
      className="buttonRow"
    >
      <a>
        <button
          className="button"
          type="submit"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-file-upload fa-w-12 fa-2x iconCenter"
            data-icon="file-upload"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 384 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224 136V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zm65.18 216.01H224v80c0 8.84-7.16 16-16 16h-32c-8.84 0-16-7.16-16-16v-80H94.82c-14.28 0-21.41-17.29-11.27-27.36l96.42-95.7c6.65-6.61 17.39-6.61 24.04 0l96.42 95.7c10.15 10.07 3.03 27.36-11.25 27.36zM377 105L279.1 7c-4.5-4.5-10.6-7-17-7H256v128h128v-6.1c0-6.3-2.5-12.4-7-16.9z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <span>
            Send Message
          </span>
        </button>
      </a>
      <a
        href="mailto:robert@kochie.io?Subject=Hi%20I'm%20&body="
      >
        <button
          className="button"
          type="button"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-envelope fa-w-16 fa-2x iconCenter"
            data-icon="envelope"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <span>
            Email
          </span>
        </button>
      </a>
      <a
        download="Robert_Koch"
        href="/Robert_Koch.pdf"
      >
        <button
          className="button"
          type="button"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-file-download fa-w-12 fa-2x iconCenter"
            data-icon="file-download"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 384 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224 136V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zm76.45 211.36l-96.42 95.7c-6.65 6.61-17.39 6.61-24.04 0l-96.42-95.7C73.42 337.29 80.54 320 94.82 320H160v-80c0-8.84 7.16-16 16-16h32c8.84 0 16 7.16 16 16v80h65.18c14.28 0 21.4 17.29 11.27 27.36zM377 105L279.1 7c-4.5-4.5-10.6-7-17-7H256v128h128v-6.1c0-6.3-2.5-12.4-7-16.9z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <span>
            Download Resume
          </span>
        </button>
      </a>
    </div>
  </form>
</div>
`;
